profile/Makefile

```diff
@@ -1,7 +1,7 @@
 EXTRAGOARGS:=-v -race -cover
 
 test:
-	sudo env "PATH=$(PATH)" go test ./ $(EXTRAGOARGS)
+	sudo env "PATH=$(PATH)" go test ./plotter*.go $(EXTRAGOARGS)
```

Comment:
What is the impact on the coverage of this change?

Reply:
The whole `profiler.go` would not be covered,  I do not thing this is a good idea. 
I'm working on adding a self-hosted runner.
Will delete this PR if that works